versions.mk

```diff
@@ -3,6 +3,8 @@ CONTROLLER_TOOLS_VERSION := v0.13.0
 # ENVTEST_K8S_VERSION refers to the version of kubebuilder assets to be downloaded by envtest binary.
 # NOTE: the suffix .x means wildcard match so specifying the latest patch version.
 ENVTEST_K8S_VERSION := 1.28.x
+# ENVTEST_VERSION is usually latest, but might need to be pinned from time to time.
```

Comment:
```suggestion
# ENVTEST_VERSION is usually latest, but might need to be pinned from time to time.
# Version pinning is needed due to version incompatibility between controller-runtime and setup-envtest.
# For more information: https://github.com/kubernetes-sigs/controller-runtime/issues/2744
```

Reply:
Thanks for your feedback! I updated my patch. Could you review this again?